docs/src/Touchable.doc.js

```diff
@@ -73,51 +73,77 @@ card(
 class TouchableExample extends React.Component {
 constructor(props) {
   super(props);
-  this.state = { clicks: 0 };
-  this.handleTouch = this._handleTouch.bind(this);
+  this.state = { imageClicks: 0, inlineClicks: 0 };
+  this.handleImageTouch = this._handleImageTouch.bind(this);
+  this.handleInlineTouch = this._handleInlineTouch.bind(this);
```

Comment:
@mminute instead of adding it to this example, we probably want to create a new one instead. (We're trying to keep these examples as small as possible)